CONTRIBUTING.md

```diff
@@ -5,28 +5,25 @@
 Fork and clone this repository, then run
 
 ```bash
-npm install
+yarn
 ```
 
 Run tests:
 
 ```bash
-npm test
+yarn test
 ```
 
 ## Code
 
 This repository is a monorepo which means that it contains multiple npm packages.
 
-Running `npm install` in the repository's root directory will install the dependencies of all packages and link the packages that depend on each other.
+Running `yarn` in the repository's root directory will install the dependencies of all packages and link the packages that depend on each other.
```

Comment:
See [this comment](https://github.com/andywer/webpack-blocks/pull/273/files#diff-6a3371457528722a734f3c51d9238c13R8)